src/Entity/CitizenProject.php

```diff
@@ -385,4 +387,30 @@ public function getCreator(): ?Adherent
     {
         return $this->creator;
     }
+
+    public function getSkills(): Collection
+    {
+        return $this->skills;
+    }
+
+    public function setSkills(iterable $skills): void
+    {
+        foreach ($skills as $skill) {
+            if ($skill instanceof CitizenProjectSkill) {
```

Comment:
I'm not sure this check is needed. And if it is, it would be better to throw an `\InvalidArgument` when this is not true.